db/health/migrate/20230504143929_renewed_cp1_careplans.rb

```diff
@@ -0,0 +1,27 @@
+class RenewedCp1Careplans < ActiveRecord::Migration[6.1]
+  def up
+    cp2_date = (Health::PatientReferral::CP_2_REFERRAL_DATE ..)
+
+    # Only careplans touched after the CP2 date can have been renewed...
+    careplans = Health::Careplan.where(updated_at: cp2_date ..).
```

Comment:
cp2_date is already a range, this is weird.  I think you meant:
```suggestion
    careplans = Health::Careplan.where(updated_at: cp2_date).
```

Reply:
Just to confirm the logic
The care plan must have been:
1. Updated after the particular date
2. Signed with the year prior to the date
3. approved after the date.

#2 doesn't feel right to me.

Reply:
Oops -- I decided I needed the date not the range, and didn't fix cp2_date. Thanks!

Reply:
Why does the PCTP need to be updated and approved _after_ the signature?

Reply:
The logic is supposed to be:
1. The careplan was updated after CP2 started (so, they have started renewing it)
2. The careplan was signed before CP2 started, but in the last year, (so it was a completed CP1 careplan)
3. The careplan was approved, which means the logic that will handle renewals after deployment would not be triggered, so create the missing QA.

Reply:
Ok, that seems to track

Reply:
MH is requiring them to renew any current CP1 careplans. This is basically, a mechanical process of having the CC and RN look over it, and send it to the PCP.